content/sensu-go/5.18/guides/clustering.md

```diff
@@ -37,6 +37,8 @@ You can configure a Sensu cluster in a couple different ways &mdash; we'll show
 
 We also recommend using stable platforms to support your etcd instances (see [etcd's supported platforms][5]).
 
+_**IMPORTANT**: If any of your cluster members was started **before** being clustered, remove that member's path to state storage (`/var/lib/sensu/sensu-backend/etcd`) before continuing with configuration._
```

Comment:
```suggestion
_**IMPORTANT**: If any of your cluster members were started **before** being configured to join a cluster, that member will have persisted its prior configuration to disk. In order to join such a member to a cluster, you must first remove that member's etcd data by stopping sensu-backend and deleting the contents of `/var/lib/sensu/sensu-backend/etcd` before proceeding with configuration._
```

Reply:
How does this sound:

"If a cluster member is started **before** it is configured to join a cluster, the member will persist its prior configuration to disk. For this reason, you must remove any previously started member's etcd data by stopping sensu-backend and deleting the contents of `/var/lib/sensu/sensu-backend/etcd` before proceeding with cluster configuration."

Reply:
...and if that note is good, should this be propagated back to 5.15?

Reply:
@hillaryfraley I think your suggestion works, and I think this change is appropriate for all currently supported versions.

Reply:
Thank you!